### ISSUE TITLE
"M" at the end of the Subscribers button.

### ISSUE THREAD

**Issue by kwan-vini** [lightbeam24/CustomTube#36]
![image](https://github.com/lightbeam24/CustomTube/assets/118130060/171edc9f-85b9-4488-8e01-85070e8e8400)

Affects all layouts. Might be something langauge related.

**Comment by lightbeam24** [comment]
Yep, that's language related all right. Supporting multiple languages is something I would like to do in the future, but it will be a pretty big task, and I only know English, so I can't say when this will be fixed. Could you tell me what language this is so I can possibly add support for it later?

**Comment by kwan-vini** [comment]
> Yep, that's language related all right. Supporting multiple languages is something I would like to do in the future, but it will be a pretty big task, and I only know English, so I can't say when this will be fixed. Could you tell me what language this is so I can possibly add support for it later?

Yeah, developing takes time and i know it gets tiring after a while. Just take your time, don't stress yourself :)

About the language, it's Portuguese (more specifically the Brazilian variation.
